src/mds/Server.cc

```diff
@@ -244,14 +244,14 @@ void Server::dispatch(const cref_t<Message> &m)
     handle_client_reconnect(ref_cast<MClientReconnect>(m));
     return;
   }
-
+  bool sessionclosed_isok = g_conf().get_val<bool>("mds_replay_unsafe_with_closed_session");
```

Comment:
This should be cached by the Server. See `Server::handle_conf_change`.

Reply:
> This should be cached by the Server. See `Server::handle_conf_change`.

done